download.md

```diff
@@ -4,9 +4,66 @@ title: Downloads
 permalink: /downloads/
 ---
 
-Once the first version of PELUX is released you will be able to find download
-links to the images, SDK and ADK, etc.
+# PELUX 1.0
+As of 2018-01-12, PELUX 1.0 has been released!
 
+## Main features
+ - Derived from poky
+ - Support for [Raspberry PI 3](https://www.raspberrypi.org/products/raspberry-pi-3-model-b/) and [Intel NUC](https://www.intel.com/content/www/us/en/products/boards-kits/nuc.html)
+ - Platform Development Kit (PDK)
+ - Software Development Kit (SDK)
+ - 2 flavours: with and without [Qt Automotive Suite](https://www1.qt.io/qt-automotive-suite/)
+ - DLT integration
+ - BT integration
+ - Publicly accessible CI
+
+## Metadata
+Version number: 1.0
+Yocto release: morty and pyro
+Qt version: 5.9.3
+
+## Changelog
+This is the initial release of PELUX.
+
+## Known issues
```

Comment:
Could you make this in to a bullet list? We should have open github issues we can assign to these ideally or if applicable. I can do that later on.

Reply:
It is a bullet list :)